draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -175,9 +175,14 @@ When there is no user consent, the following filtering should be done to prevent
 Generated names reuse
 ----------------------------
 
-Dynamically generated names can be used to track users if used too often.
-Conversely, registering too many names will also generate useless processing.
-The proposed rule is to create and register a new generated name for a given IP address on a per execution context.
+It is important that use of registered mDNS hostnames is limited in time and/or scope. Indefinitely reusing the same mDNS hostname candidate would provide applications an even more reliable tracking mechanism than the private IP addresses that this specification is designed to hide. The use of registered mDNS hostnames SHOULD be scoped by origin, and have the lifetime of the page.
+
+Fingerprinting
+----------------------------
+
+If the generate mDNS hostname candidate does not follow the pattern described, the exposed candidate might be used for fingerprinting.
+
+If there are multiple host candidates with different IP addresses, IPv4 and/or IPv6, each results in a separate mDNS hostname candidate. The number of mDNS hostname candidates can provide a fingerprinting dimension. If so desired an ICE agent MAY expose additional mDNS hostname candidates that are not registered.
```

Comment:
Seems to good to have this issue there.
I am not enthusiastic about exposing more candidates, let's continue digging there further.